src/odin/maneuver.cc

```diff
@@ -1279,7 +1279,7 @@ std::string Maneuver::ToString() const {
   for (const auto& guidance_view : guidance_views_) {
     man_str += "[";
     man_str += " data_id=" + guidance_view.data_id();
-    man_str += " type=" + guidance_view.type();
+    // man_str += " type=" + guidance_view.type();
```

Comment:
@dgearhart do you want to write a function that converts this to a string or is it ok for us to just omit it?

Reply:
we can also just use `std:to_string` on it? (it might be helpful to have the guidance view type logged)

Reply:
agreed - we can just do `std:to_string` for now and add function later

Reply:
Pushed change, all green again